docs/signature-layout.md

```diff
@@ -0,0 +1,29 @@
+# Signature File Layout
+
+A common file layout for storing and serving signatures provides a consistent way to reference image signatures. Signatures on a filesystem or a web server shall use this common layout. Signatures stored in a REST API are not required to use this common layout.
+
+## Specification
+
+**SCHEME://[URI]/PATH_PREFIX/REGISTRY/REPOSITORY/IMAGE@MANIFEST_DIGEST/signature-INT**
+
+* **SCHEME**: The transport scheme, e.g. **file://** or **https://**
+* **URI**: For remote transport schemes, the signature server URI, e.g. **example.com**
```

Comment:
The definition of a URI from RFC 3986 is quite different. https://tools.ietf.org/html/rfc3986